### DISCUSSION TITLE
一分钟读论文：《要不要上 TypeScript？GitHub 上 JS 和 TS 应用软件质量的系统比较》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#16] [Announcements]
# 一分钟读论文：《要不要上 TypeScript？GitHub 上 JS 和 TS 应用软件质量的系统比较》

你的团队是否还在纠结要不要将项目改成 TypeScript 呢？改成 TypeScript 应用的 bug 能得到收敛吗？德国斯图加特大学软件工程研究所的这篇新论文《To Type or Not to Type? A Systematic Comparison of the Software Quality of JavaScript and TypeScript Applications on GitHub》能给你答案。

https://unbug.github.io/On-A-Systematic-Comparison-of-the-Software-Quality-of-JavaScript-and-TypeScript-Applications-on-GitHub/

**Comment by unbug** [comment]
TypeScript 的副作用就是修个 bug 越来越耗时了
